lib/install/management/store_files.go

```diff
@@ -126,7 +126,7 @@ func (d *Dispatcher) deleteDatastoreFiles(ds *object.Datastore, path string, for
 	}
 
 	m := object.NewFileManager(ds.Client())
-	if d.isVSAN(ds) {
+	if d.geVC65() || d.isVSAN(ds) {
```

Comment:
1. Why is the caller responsible for knowing the exact model by which you have to delete files?
2. Why even bother with separate paths (this is purely an optimization)

Can we have d.deleteFiles(m, ds, dsPath) - if we really have to have the optimization, the check should be inside that function.